test/packages/good_content/manifest.yml

```diff
@@ -14,6 +13,8 @@ conditions:
     version: '^8.16.0'
   elastic:
     subscription: 'basic'
+    capabilities:
+      - security
```

Comment:
As there are `kibana/security_rule/*.json` assets/files, it is required to add this capability.

Reply:
Should we also add a semantic validation rule that checks this?

Reply:
There is already in place one semantic validation rule for that. It failed for me locally, that's why I added here.

Reply:
This is the semantic validation rule for this scenario:
https://github.com/elastic/package-spec/blob/622b3a8f000ea7f40d5c2ad64801b94be4fa2522/code/go/internal/validator/semantic/validate_capabilities_required.go#L26